lstchain/calib/camera/flatfield.py

```diff
@@ -74,6 +79,15 @@ def __init__(self, **kwargs):
         self.arrival_times = None  # arrival time per event in sample
         self.sample_masked_pixels = None  # masked pixels per event in sample
 
+        # declare time calibrator if correction file exist
+        if os.path.exists(self.time_calibration_path):
+            self.time_corrector = PulseTimeCorrection(
+                calib_file_path = self.time_calibration_path)
+        else:
+            self.time_corrector = None
+            self.log.info(f"File {self.time_calibration_path} not found. No drs4 time corrections")
```

Comment:
I think this should not be silently ignored and for sure not be "info".

If the user gave a calibration path that does not exist, I think it would be better to fail loudly and immediately.

So this should be `raise IOError('Time calibration file {self.time_calibration_path' not found')`, or just use the `exists=True` option of the `Path` traitlet, see above.

Reply:
This is because at the beginning not all nights had the time calibration file. Now we can indeed always ask for it. I will add the raise IOError because I get problems with the Path

Reply:
What kind of problems? You should use Path. If you have problems with it, let's solve them instead of doing something else

Reply:
Does `timecalibration_path = Path(None, exists=True, directory_ok=False)` not work for you?

Reply:
I want to use it as trailet, how do you add the help? (it didn't like it)

Reply:
Yes, sorry. Support for the help and other standard traitlets args was only added after 0.7.

I would to suggest to leave it as unicode then und wait with `Path` for the ctapipe 0.8 switch